### ISSUE TITLE
Upgrade Ignition Version

### ISSUE THREAD

**Issue by jlbcontrols** [jlbcontrols/Flintium#203]
Upgrade Flintium's Ignition version to latest stable Ignition LTS version

-Use the standard Ignition upgrade process
-Edit README and docker files with the latest version

**Comment by ArnaudDeClerck** [comment]
i can update the actual version to 8.1.8 but i don't have that much experience in git and docker

**Comment by jlbcontrols** [comment]
No problem, I can take care of the docker and readme changes after you work through the upgrade.

To get started, read through the [How to make a pull request](https://github.com/jlbcontrols/Flintium/wiki/Contributing#how-to-make-a-pull-request) section of the Flintium Wiki. Try to follow the instructions, and let me know when you run into problems, or have any questions. We can set up a screen share if you'd like as well.

**Comment by ArnaudDeClerck** [comment]
@jlbcontrols project is updated to 8.1.9 do you need a gateway backup ?
I follow the developpement step of the readme section

**Comment by jlbcontrols** [comment]
Great! No gateway backup is needed for now. We are only committing any changes that have been automatically applied to the project during the upgrade process (if any).
Now you just use the following steps (if you haven't already):
checkout a new branch
`git checkout -b "8.1.9"`
in the project folder, add and commit the changes:
`git add .`
`git commit -m "upgrade project to 8.1.9"`
push to your remote fork repository
`git push origin 8.1.9`
then create a pull request to bring the changes into this repository using Github

**Comment by ArnaudDeClerck** [comment]
I don't have the permission to push on this branch

**Comment by jlbcontrols** [comment]
Yes, you'll to create a pull request instead. The top of your Git repository should look something like this:
![image](https://user-images.githubusercontent.com/46946127/129577350-5e432368-a449-43ad-a193-2fc5a4530619.png)
From there, you can click "Contribute" to create the pull request.

**Comment by ArnaudDeClerck** [comment]
Hi,
i don't see the branch 8.1.9 in the flintium github is dit normal? Do i need to make my one test git and then make a pull request from my git to the flintium one ?

**Comment by jlbcontrols** [comment]
Yes that is correct. You need to create the branch in your "fork", then create the pull request.

**Comment by ArnaudDeClerck** [comment]
Ok thanks for the help (Sorry i'm still learning github). Thera was no file change between the fork taken from your master branch and the updated file in 8.1.9.

**Comment by jlbcontrols** [comment]
No problem! Since there are no project changes, I will reassign this Issue to me to make the README and docker changes.
